src/redist/targets/GenerateBundledVersions.targets

```diff
@@ -34,8 +34,8 @@
       <_WindowsDesktop50DefaultRuntimeFrameworkVersion>5.0.0</_WindowsDesktop50DefaultRuntimeFrameworkVersion>
       <_WindowsDesktop50RuntimePackVersion>5.0.0</_WindowsDesktop50RuntimePackVersion>
       <_WindowsDesktop50TargetingPackVersion>5.0.0</_WindowsDesktop50TargetingPackVersion>
-      <_AspNet50DefaultRuntimeFrameworkVersion>5.0.0</_AspNet50DefaultRuntimeFrameworkVersion>
-      <_AspNet50RuntimePackVersion>5.0.0</_AspNet50RuntimePackVersion>
+      <_AspNet50DefaultRuntimeFrameworkVersion>5.0.1</_AspNet50DefaultRuntimeFrameworkVersion>
```

Comment:
Is this actually used anywhere now?  Same for the other _*50DefaultRuntimeFrameworkVersion values

Reply:
Looks like it isn't, I'll remove these once we get clarity on what to do about `MicrosoftWindowsSDKNETRef10_0_17763PackageVersion`